tests/test_getRegionTimeFromVisit.py

```diff
@@ -23,15 +23,14 @@
 import tempfile
 
 import astropy.time
-import pandas as pd
 
-import lsst.utils.tests
-from lsst.sphgeom import ConvexPolygon, UnitVector3d
+from lsst.afw.table import SourceCatalog
 import lsst.daf.butler
 import lsst.daf.butler.tests as butlerTests
 import lsst.pipe.base.testUtils as pipeTests
-
 from lsst.pipe.tasks.getRegionTimeFromVisit import GetRegionTimeFromVisitTask
+from lsst.sphgeom import ConvexPolygon, UnitVector3d
+import lsst.utils.tests
```

Comment:
In the future, please [put this kind of reorganization on its own commit](https://developer.lsst.io/work/flow.html#git-commit-organization-best-practices).